README.md

```diff
@@ -173,6 +173,56 @@ To remove the entry from the credentials file, use `oras logout`:
 oras logout <registry-ip>:8443
 ```
 
+#### Example pull/push using with HTTP Docker registry
+
+You want to pull or push the HTTP Docker registry. `oras` support `--plain-http` flag to pull or push.
+
+The `--plain-http` flag mean that you want to use http instead of https to connect the Docker registry.
+
+First, create a valid htpasswd file (must use `-B` for bcrypt):
+
+```
+htpasswd -cB -b auth.htpasswd myuser mypass
+```
+
+Next, start a registry using that file for auth and listen the `0.0.0.0` address:
+
+```
+docker run -it --rm -p 5000:5000 \
+    -v $(pwd)/auth.htpasswd:/etc/docker/registry/auth.htpasswd \
+    -e REGISTRY_AUTH="{htpasswd: {realm: localhost, path: /etc/docker/registry/auth.htpasswd}}" \
+    -e REGISTRY_HTTP_ADDR=0.0.0.0:5000 \
+    registry
+```
+
+In a new window, login with `oras` using the ip address not localhost:
+
+```
+oras login -u myuser -p mypass --insecure <registry-ip>:5000
+```
+
+You will notice a new entry for `<registry-ip>:5000` appear in `~/.docker/config.json`.
+
+Then you can pull files from the registry or push files to the registry.
+
+To push single file to this registry:
+
+```
+oras push --insecure <registry-ip>:5000/library/hello:latest hi.txt --plain-http
```

Comment:
same here - I would expect to only ever use one or the other (`--insecure` or `--plain-http`)